docs/_site/build/gh-pages/src/project-loader.js

```diff
@@ -0,0 +1,79 @@
+import { LitElement, html } from "../node_modules/@polymer/lit-element/lit-element.js";
+export class ProjectLoader extends LitElement {
+  static get properties() {
+    return {
+      folder: {
+        type: String
+      },
+      project: {
+        type: Object
+      }
+    };
+  }
+
+  render() {
+    return html``;
+  }
+
+  firstUpdated() {
+    this.project = {};
+    this.loadProject();
+  }
+
+  loadProject() {
```

Comment:
async loadProject and then use of async/await would make the below easier to read